docs/Getting_Involved/Newcomers.rst

```diff
@@ -0,0 +1,116 @@
+Welcome to the Newcomers guide!
+===============================
+
+This guide will help you to get involved and teach you how to make your first
+contribution to coala!
+
+Meet the community!
+-------------------
+
+To get started, the first way is to meet the community. We use gitter to
+communicate, and there the helpful community will guide you.
+
+::
+
+    https://gitter.im/coala-analyzer/coala
+
+
+Congratulations! You are now part of our community.
+
+Start working
+-------------
+
+Let us know that you are willing to contribute and you will be added to our
+organization on github. This is your first step towards contributing.
+The invitation will be sent upon mail and you will have to accept it to join.
+
+Now that you are part of our organization, you can start working on issues.
+
+::
+
+    https://github.com/coala-analyzer
+
+
+Get help with git
+-----------------
+
+We use github to manage our repository. If you're not familiar with git, we
+strongly recommend following a tutorial, such as:
+
+::
+
+    https://try.github.io/levels/1/challenges/1
+
+
+If there's anything unclear, or you are encountering problems, feel free
+to contact us on git, and we will help you!
+
+Start working
+-------------
+
+Now is the time to pick an issue.
+
+::
+
+    https://github.com/coala-analyzer/coala/issues
```

Comment:
not this link, we have a link that lists newcomer issues from all repos, it's important to always use that one